src/Sylius/Bundle/CoreBundle/Controller/ProductController.php

```diff
@@ -283,4 +316,38 @@ private function renderResults(
 
         return $this->handleView($view);
     }
+
+    private function renderArchetypeResults(
+        ArchetypeInterface $archetype,
+        Pagerfanta $results,
+        $template,
+        $page,
+        $facets = null,
+        $facetTags = null,
+        $filters = null,
+        $searchTerm = null,
+        $searchParam = null,
+        $requestMethod = null
+    )
+    {
+        $results->setCurrentPage($page, true, true);
+        $results->setMaxPerPage($this->config->getPaginationMaxPerPage());
+
+        $view = $this
+            ->view()
+            ->setTemplate($this->config->getTemplate($template))
+            ->setData(array(
+                'archetype'    => $archetype,
```

Comment:
either you should align the key value together with the whole array block (don't know if it's allowed in code style in Sylius), or you should do a single space before and after the `=>` sign, but not a random number of space :)